packages/stylelint-config-ckeditor5/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "stylelint-config-ckeditor5",
+  "version": "2.0.0",
```

Comment:
Why do we start with 2.0.0 version instead of 1.0.0?

Reply:
```suggestion
  "version": "1.0.0",
```